src/common/k8s-api/kube-api.ts

```diff
@@ -643,11 +643,12 @@ export class KubeApi<
       clearTimeout(timedRetry);
     });
 
+    setTimeoutFor(abortController, 60 * 1000);
```

Comment:
Could you explain how this matches the previous behaviour.. it is a bit unclear to me (we had 10x timeout before).

Reply:
Good catch, it should be the same.